### DISCUSSION TITLE
Compared with ballista?

### DISCUSSION THREAD

**Discussion by smallzhongfeng** [databendlabs/databend#13477] [Q&A] [answered]
Since it is also a distributed query engine, have you compared the performance with ballista, or conducted a TPCH performance test with spark?

**Comment by bohutang** [comment, marked as answer]
No.

TPCH-SF100, with DuckDB:
![image](https://github.com/datafuselabs/databend/assets/172204/9d3eb7b0-6db8-4496-b7a3-38ac1eb0ed7a)

With Snowflake:
![image](https://github.com/datafuselabs/databend/assets/172204/5a836501-8f43-4a08-8324-8df0d5ef1423)

**Comment by smallzhongfeng** [comment]
Got it, looks good.👍
